include/c_types/contracted_rt.h

```diff
@@ -0,0 +1,71 @@
+/*PGR-GNU*****************************************************************
+File: pgr_types.h
+
+Copyright (c) 2015 Celia Virginia Vergara Castillo
+Mail: vicky_vergara@hotmail.com
```

Comment:
Ok, the design is by me, but the change is by you so lets make this:

Copyright (c) 2017 pgRouting developers
Mail: project@pgrouting.org